Application/Commands/Enquiry/Build/AddEnquiryCommand.cs

```diff
@@ -335,32 +407,52 @@ private async Task HandleDbUpdateException(DbUpdateException ex, Domain.Enquiry
                 throw ex;
             }
         }
+        return updatedEnquiry;
     }
 
-    private async Task TrySendEnquirySubmittedConfirmationToEnquirerEmail(Domain.Enquiry enquiry,
-        NotificationsRecipientDto getEnquirySubmittedConfirmationToEnquirerNotificationsRecipient)
+    private async Task ProcessEmailsAsync(Domain.Enquiry enquiry)
     {
         try
         {
-            await _notificationsClientService.SendEmailAsync(
-                getEnquirySubmittedConfirmationToEnquirerNotificationsRecipient,
-                EmailTemplateType.EnquirySubmittedConfirmationToEnquirer);
+            await _processEmailsService.SendEmailAsync(enquiry.EnquirerEnquirySubmittedEmailLog.Id);
         }
         catch (Exception)
         {
             await CleanUpData(enquiry);
             throw;
         }
+
+        try
+        {
+            if (!_sendTuitionPartnerEmailsWhenEnquirerDelivered)
+            {
+                var emailLogIds = enquiry.TuitionPartnerEnquiry.Select(x => x.TuitionPartnerEnquirySubmittedEmailLogId).ToArray();
+                await _processEmailsService.SendEmailsAsync(emailLogIds!);
+            }
+        }
+        catch { } //We suppress the exceptions here since we want the user to get the confirmation page, errors are logged in the services
```

Comment:
Seeing the following error on my IDE "Nested blocks of code should not be left empty".

We should put the comment inside of the catch block.

For example:
```

catch
{
    // We suppress the exceptions here since we want the user to get the confirmation page, errors are logged in the services
} 
```

Reply:
I think that this is specific to your IDE (isn't an issue in VS).  There are 5 instances of exceptions being handled in this way (some are live) throughout the code.  I have updated all 5.